automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -52,46 +52,52 @@ class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with
   "A user whose free trial status is" - {
 
     "Blank" - {
-      "should not see the free trial banner" in withWebDriver { implicit driver =>
-        withSignIn(testUser) { _ =>
-          await ready new WorkspaceListPage()
-          val bannerTitleElement = Label(TestId("trial-banner-title")) // TODO: Define elements in page class
-          bannerTitleElement.isVisible shouldBe false
+      "should not see the free trial banner" in {
+        withWebDriver { implicit driver =>
+          withSignIn(testUser) { _ =>
+            await ready new WorkspaceListPage()
+            val bannerTitleElement = Label(TestId("trial-banner-title")) // TODO: Define elements in page class
+            bannerTitleElement.isVisible shouldBe false
+          }
         }
       }
     }
 
     "Enabled" - {
-      "should be able to see the free trial banner, enroll and get terminated" in withWebDriver { implicit driver =>
+      "should be able to see the free trial banner, enroll and get terminated" in {
         setUpEnabledUserAndProject(testUser)
         val trialAuthToken = TrialBillingAccountAuthToken()
         api.trial.reportTrialProjects().foreach { x =>
-          register cleanUp Rawls.admin.deleteBillingProject(x.name, UserInfo(OAuth2BearerToken(trialAuthToken.value), WorkbenchUserId("0"), WorkbenchEmail("doesnt@matter.com"), 3600))(UserPool.chooseAdmin.makeAuthToken())
+          register cleanUp Try(Rawls.admin.deleteBillingProject(x.name, UserInfo(OAuth2BearerToken(trialAuthToken.value),
+            WorkbenchUserId("0"), WorkbenchEmail("doesnt@matter.com"), 3600))(UserPool.chooseAdmin.makeAuthToken())).recover {
+              case ex: RestException => logger.warn(s"RestException occurred in Rawls.admin.deleteBillingProject(${x.name})", ex)
+          }
         }
+        withWebDriver { implicit driver =>
```

Comment:
More importantly because, once the browser is open, Selenium can fail the test after detecting long inactivity with the window. In other words, we may still be happily waiting for billing project creation but Selenium will fail us anyway. Better to do long-running test setup before opening the window. Good change. 👍